map/src/main/kotlin/edu/artic/map/MapFragment.kt

```diff
@@ -9,22 +9,43 @@ import com.bumptech.glide.Glide
 import com.bumptech.glide.request.RequestOptions
 import com.bumptech.glide.request.target.SimpleTarget
 import com.bumptech.glide.request.transition.Transition
-import com.fuzz.rx.*
+import com.fuzz.rx.bindTo
+import com.fuzz.rx.defaultThrottle
+import com.fuzz.rx.disposedBy
+import com.fuzz.rx.filterFlatMap
+import com.fuzz.rx.filterValue
+import com.fuzz.rx.mapOptional
 import com.google.android.gms.maps.CameraUpdateFactory
 import com.google.android.gms.maps.GoogleMap
 import com.google.android.gms.maps.MapsInitializer
-import com.google.android.gms.maps.model.*
+import com.google.android.gms.maps.model.BitmapDescriptor
+import com.google.android.gms.maps.model.BitmapDescriptorFactory
+import com.google.android.gms.maps.model.CameraPosition
+import com.google.android.gms.maps.model.GroundOverlay
+import com.google.android.gms.maps.model.GroundOverlayOptions
+import com.google.android.gms.maps.model.LatLng
+import com.google.android.gms.maps.model.LatLngBounds
+import com.google.android.gms.maps.model.MapStyleOptions
+import com.google.android.gms.maps.model.Marker
+import com.google.android.gms.maps.model.MarkerOptions
```

Comment:
Something seems odd about your import settings - is the IDE automatically replacing the `*` with explicit imports?

Reply:
yeah. It turned off `*` imports. Not sure if thats an issue, something to discuss later. I think for now ill keep it as is and we can tweak that setting in the project so its the same for everyone.